controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
+ */
+module.exports.readAll = async (req, res) => {
+  const allGenres = await Genre.find({});
+
+  if (res) res.json(allGenres);
+  else return allGenres;
```

Comment:
This should return the ```res.json```, which also makes it so you don't actually need the if-else, and satisfies that the arrow function always returns something:

```suggestion
if (res) { 
  return res.json(allGenres); 
}
return allGenres;
```

Reply:
Previously, the seeder would call this, but since that is not longer the case, this will always ```return res.json```.